hamilton/dataflows/__init__.py

```diff
@@ -498,10 +498,10 @@ def are_py_dependencies_satisfied(dataflow, user=None, version="latest"):
             else:
                 package_name = line
                 required_version = None
-            required_version  # here for now...
+            required_version  # noqa here for now...
```

Comment:
Just curious - what is the purpose of these statements?

Reply:
below I have commented out code that would use it. I never got to turning it on completely.